### PR TITLE
implement epic 2

### PR DESCRIPTION
**Pull request by kolya2po**

### CODE REVIEW

File: GameStore.DAL/Entities/Game.cs [hunk 1]
```diff
@@ -9,5 +9,6 @@ public class Game : BaseEntity
         public decimal Price { get; set; }
         public IEnumerable<GameGenre> Genres { get; set; }
         public string Description { get; set; }
+        public int AuthorId { get; set; }
```
Comment (sivokon) [inline GameStore.DAL/Entities/Game.cs:12]: Is it which user added the game?

File: GameStore.BLL/Infrastructure/JwtHandler.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (sivokon) [inline GameStore.BLL/Infrastructure/JwtHandler.cs:2]: not used references

File: GameStore.BLL/Services/UsersService.cs [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            await _userManager.UpdateAsync(user);
+
+        }
+
+        public async Task SignOutAsync()
+        {
+            await _signInManager.SignOutAsync();
+        }
+
+        private string GetJwtToken()
```
Comment (sivokon) [inline GameStore.BLL/Services/UsersService.cs:109]: I would suggest to move this logic to JwtHandler, seems like it should not belong to UsersService. We can have such method in JwtHandler and current methods of JwtHandler can be made private

File: GameStore.BLL/Services/UsersService.cs [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        private static void ValidateFile(IFormFile avatar)
+        {
+            if (avatar == null)
+            {
+                throw new GameStoreException("Avatar was null.");
+            }
+
+            if (!avatar.ContentType.Contains("image"))
+            {
+                throw new InvalidFileContentTypeException();
```
Comment (sivokon) [inline GameStore.BLL/Services/UsersService.cs:128]: How this type of exception will be handled by ErrorHandlingMiddleware? Will it return InternalServerError?

File: GameStore.BLL/Models/Identity/RegistrationModel.cs [hunk 5]
```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace GameStore.BLL.Models.Identity
+{
+    public class RegistrationModel
+    {
+        [Required(ErrorMessage = "You should provide your first name.")]
```
Comment (sivokon) [inline GameStore.BLL/Models/Identity/RegistrationModel.cs:7]: It is not critical but I would suggest to change messages to something like "FirstName is require", without any appeal :)

File: GameStore.WebApi/Startup.cs [hunk 6]
```diff
@@ -23,16 +24,30 @@ public void ConfigureServices(IServiceCollection services)
...
                 app.UseDeveloperExceptionPage();
+                app.UseSwagger();
+                app.UseSwaggerUI(options =>
+                {
+                    options.SwaggerEndpoint("/swagger/v1/swagger.json", "v1");
+                    options.RoutePrefix = string.Empty;
+                });
             }
 
             app.UseStaticFiles();
```
Comment (sivokon) [inline GameStore.WebApi/Startup.cs:53]: Do we need to add UseAuthentication middleware?

File: GameStore.BLL/Services/UsersService.cs (hunk 4 above)
Comment (kolya2po) [reply GameStore.BLL/Services/UsersService.cs:128]: It's derived from GameStoreException.

File: GameStore.WebApi/Startup.cs (hunk 6 above)
Comment (kolya2po) [reply GameStore.WebApi/Startup.cs:53]: Definitely. Although not now)

**Pull request merged**